files/settings-default.cfg

```diff
@@ -174,6 +174,13 @@ buffer cache min = 14
 # to this much memory until old buffers get purged.
 buffer cache max = 16
 
+# Specifies whether to enable HRTF processing. Valid values are: -1 = auto,
+# 0 = off, 1 = on.
+hrtf enable = -1
```

Comment:
> so how does OpenAL know if a device is headphone or speaker?

The audio system can indicate what it's configured as. At least for DSound, WASAPI, and PulseAudio, it can to some degree. OpenAL Soft also has its own config option for dictating if a device should be treated as headphones or speakers, so it will Just Work if the user has it preconfigured. Though it ultimately errs on the side of caution... if it doesn't know it's headphones, it treats it as speakers and won't use HRTF unless explicitly requested to do so.